tools/functions.js

```diff
@@ -88,7 +88,7 @@ function findDependencies(startFile, class2Path, deps) {
 
         deps.push(dep);
 
-        if (dep === 'Nimiq') return;
+        if (dep === 'Nimiq' || dep === 'CONFIG') return;
```

Comment:
Why is this necessary? the CONFIG uses the `Constants` global, so I think it's fair to continue building the dependency graph for config as well.